Compile with QtCreator 3.5

I'm getting the following error when trying to compile with QtC 3.5:

```
src/dlangoutlinemodel.cpp: In member function 'virtual QStringList DlangEditor::DlangOutlineModel::mimeTypes() const':
src/dlangoutlinemodel.cpp:152:19: error: 'Utils::FileDropSupport' has not been declared
src/dlangoutlinemodel.cpp: In member function 'virtual QMimeData* DlangEditor::DlangOutlineModel::mimeData(const QModelIndexList&) const':
src/dlangoutlinemodel.cpp:157:32: error: 'FileDropMimeData' in namespace 'Utils' does not name a type
```

--- a/src/dlangoutlinemodel.cpp
+++ b/src/dlangoutlinemodel.cpp
@@ -4,6 +4,10 @@
 
 #include <utils/qtcassert.h>
 #include <utils/fileutils.h>
+#if QTCREATOR_MINOR_VERSION >= 5
+#  include <utils/dropsupport.h>
+#endif
+
 
 #include "dlangeditor.h"
 #include "dlangimagecache.h"
@@ -149,12 +153,20 @@ Qt::DropActions DlangOutlineModel::supportedDragActions() const
 
 QStringList DlangOutlineModel::mimeTypes() const
 {
+#if QTCREATOR_MINOR_VERSION < 4
     return Utils::FileDropSupport::mimeTypesForFilePaths();
+#else
+    return Utils::DropSupport::mimeTypesForFilePaths();
+#endif
 }
 
 QMimeData *DlangOutlineModel::mimeData(const QModelIndexList &indexes) const
 {
+#if QTCREATOR_MINOR_VERSION < 4
     auto mimeData = new Utils::FileDropMimeData;
+#else
+    auto mimeData = new Utils::DropMimeData;
+#endif
     foreach (const QModelIndex &index, indexes) {
         const QVariant fileName = data(index, FileNameRole);
         if (!fileName.canConvert<QString>())

